otter/test/models/test_intents.py

```diff
@@ -15,6 +19,32 @@ def test_perform(self):
                                modifier=lambda g, o: 'new state')
         dispatcher = TypeDispatcher({
             ModifyGroupState: perform_modify_group_state})
-        d = perform(dispatcher, Effect(mgs))
-        self.assertEqual(self.successResultOf(d), 'new state')
+        result = sync_perform(dispatcher, Effect(mgs))
+        self.assertEqual(result, 'new state')
         self.assertEqual(group.modify_state_values, ['new state'])
+
+
+class GetScalingGroupInfoTests(SynchronousTestCase):
+    """Tests for :obj:`GetScalingGroupInfo`."""
+    def test_perform(self):
+        """Performing returns the group, the state, and the launch config."""
+        log = mock_log()
+        state = object()
+        lc = object()
+        group = mock_group(state)
+        group.view_state.return_value = succeed(state)
+        group.view_launch_config.return_value = succeed(lc)
+
+        data = {('00', 'g1'): group}
+
+        class Store(object):
+            def get_scaling_group(s_self, _log, tenant_id, group_id):
```

Comment:
Nonblocking - this is probably overkill, but we've had problems in the past where we've changed method signatures and interfaces without changing the mocks for tests, and hence unit tests would falsely pass.  Hence the [iMock](https://github.com/rackerlabs/otter/blob/master/otter/test/utils.py#L191) and otherwise providing a spec to `Mock`.

Would it make sense to include an assertion that the mock method signature remains the same as the actual store's `get_scaling_group` signature, so it's a verified fake?

I'm also fine with just adding a tech-debt ticket - we are unlikely to change the store interface, particularly this one, any time in the near future.